combat.sk

```diff
@@ -109,3 +109,21 @@ weapons:
 
 	bow¦s = minecraft:bow
 	arrow¦s = minecraft:arrow
+
+horse armor old:
+	minecraft version = 1.12 or older
```

Comment:
This should be `1.12.2 or older`. The ".2" is important because it's the last version of 1.12 so it's what most people on 1.12 are using, and if it's not in this version condition then these aliases won't register for those users.

Reply:
DUh... How did I not think of that!?!